taskvine/src/bindings/python3/ndcctools/taskvine/dask_executor.py

```diff
@@ -122,6 +145,12 @@ def get(self, dsk, keys, *,
             self.tune("immediate-recovery", 1)
             self.tune("max-retrievals", 10)
 
+            if self.env_per_task:
+                self.poncho_file = self.declare_file(shutil.which('ponho_package_run'))
```

Comment:
Environments do not need poncho_package_run, and should be added with `add_environment` or similar.

Reply:
Also, there is a typo in poncho_package_run.